ui/pages/onboarding-flow/create-password/create-password.js

```diff
@@ -55,44 +56,57 @@ export default function CreatePassword({
       return false;
     }
 
-    if (password.length < 8) {
+    if (password.length < PASSWORD_MIN_LENGTH) {
       return false;
     }
 
     return !passwordError && !confirmPasswordError;
   }, [password, confirmPassword, passwordError, confirmPasswordError]);
 
-  const getPasswordStrengthLabel = (score, translation) => {
+  const getPasswordStrengthLabel = (isTooShort, score, translation) => {
+    if (isTooShort) {
+      return {
+        className: 'create-password__weak',
+        text: translation('passwordNotLongEnough'),
+        description: '',
+      };
+    }
     if (score >= 4) {
       return {
         className: 'create-password__strong',
         text: translation('strong'),
         description: '',
       };
-    } else if (score === 3) {
+    }
+    if (score === 3) {
       return {
         className: 'create-password__average',
         text: translation('average'),
-        description: t('passwordStrengthDescription'),
```

Comment:
I assume that mixing references to `t` and `translation` here was not intentional.

Reply:
@legobeat It is not intentional for this references. I must forgot to add `translation` instead of `t`. Good catch! 😄

Reply:
wait what is is translation? `t` is coming from `useI18nContext()` on line 40, which we use to hook into our translation files? @legobeat

Reply:
@adonesky1 They are&were identical here so this is a non-functional change.

https://github.com/MetaMask/metamask-extension/pull/16902/files#diff-a161d5f7ee5a8ded49415fd70e45e18fa4f24dafd1e49375d77365b6cb5c84adR102

Reply:
I believe our `yarn verify-locales` routine relies on the use of `t()`, so I think it makes sense to keep `t()`.  Also for consistency.

Reply:
This is now done in afc438ed8e3ff24434c39fb9123e1eccc2c2e3f9